src/routes/docs/products/auth/mfa/+page.markdoc

```diff
@@ -0,0 +1,1018 @@
+
+{% multicode %}
+```client-web
+const challenge = await account.createMfaChallenge(
+        'phone'                       // factor
+    );
+
+// Save the challenge ID to complete the challenge later
+const challengeId = challenge.$id;
+```
+
+```client-flutter
+Future result = account.createMfaChallenge(
+factor: 'phone',
+);
+
+result.then((response) {
+    print(response);
+    // Save the challenge ID to complete the challenge later
+    var challengeId = response.$id;
+}).catchError((error) {
+    print(error.response);
+});
+```
+
+```client-apple
+let mfaChallenge = try await account.createMfaChallenge(
+  factor: "phone"
+)
+// Save the challenge ID to complete the challenge later
+let challengeId = mfaChallenge.id
+```
+
+```client-android-kotlin
+val response = account.createMfaChallenge(
+    factor = "phone"
+)
+// Save the challenge ID to complete the challenge later
+val challengeId = response.id
+```
+{% /multicode %}
+{% /tabsitem %}
+
+{% tabsitem #totp title="TOTP" %}
+
+Initiate a challenge for users to complete using an authenticator app.
+
+{% multicode %}
+```client-web
+const challenge = await account.createMfaChallenge(
+        'totp'                       // factor
+    );
+
+// Save the challenge ID to complete the challenge later
+const challengeId = challenge.$id;
+```
+
+```client-flutter
+Future result = account.createMfaChallenge(
+factor: 'totp',
+);
+
+result.then((response) {
+    print(response);
+    // Save the challenge ID to complete the challenge later
+    var challengeId = response.$id;
+}).catchError((error) {
+    print(error.response);
+});
+```
+
+```client-apple
+let mfaChallenge = try await account.createMfaChallenge(
+  factor: "totp"
+)
+// Save the challenge ID to complete the challenge later
+let challengeId = mfaChallenge.id
+```
+
+```client-android-kotlin
+val response = account.createMfaChallenge(
+    factor = "totp"
+)
+// Save the challenge ID to complete the challenge later
+val challengeId = response.id
+```
+{% /multicode %}
+{% /tabsitem %}
+{% /tabs %}
+{% /section %}
+
+{% section #complete-challenge step=8 title="Complete challenge" %}
+Once the user receives the challenge code, you can pass the code back to Appwrite to complete the challenge.
+
+{% multicode %}
+```client-web
+const response = await account.updateMfaChallenge(
+        '<CHALLENGE_ID>',            // challengeId
+        '<OTP>'                      // otp
+    );
+```
+
+```client-flutter
+Future result = account.updateMfaChallenge(
+    challengeId: '<CHALLENGE_ID>',
+    otp: '<OTP>',
+);
+
+result.then((response) {
+    print(response);
+}).catchError((error) {
+    print(error.response);
+});
+```
+
+```client-apple
+val response = account.updateMfaChallenge(
+    challengeId = "<CHALLENGE_ID>",
+    otp = "<OTP>"
+)
+```
+
+```client-android-kotlin
+let result = try await account.updateMfaChallenge(
+  challengeId: "<CHALLENGE_ID>",
+  otp: "<OTP>"
+)
+```
+{% /multicode %}
+{% /section %}
```

Comment:
Lets add some final words here. Something along lines `user is now authenticated and all requests will be authorized. You can confirm this by running `account.get()` again ,and seeing user details instead of exception now.

Reply:
Done